monodoc/Test/Monodoc/HelpSourceTests.cs

```diff
@@ -67,7 +67,8 @@ public bool Generate (HelpSource hs, string id, Dictionary<string, string> conte
 		[Test]
 		public void ReachabilityTest ()
 		{
-			var rootTree = RootTree.LoadTree (Path.GetFullPath (BaseDir), false);
+			string path = Path.GetFullPath(Path.Combine(Path.GetDirectoryName(this.GetType().Assembly.Location), BaseDir));
```

Comment:
1. Why avoid `var`?
2. Convention is space before `(`:

        var path = Path.GetFullPath (Path.Combine (Path.GetDirectoryName (this.GetType ().Assembly.Location), BaseDir));